app/viewmodels/employment/EmployerInformation.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.employment
```

Comment:
Having already the package `models`, maybe we can add a subpackage there for `views`?

In state-benefits for example, we have it as `pages` inside `models`. I don't mind much which final name it is, but it feels worth moving inside`models`

Reply:
For reference: https://github.com/hmrc/income-tax-state-benefits-frontend/tree/main/app/models/pages

Reply:
possibly, maybe let's have a team discussion about this and make a decision together?